package.json

```diff
@@ -0,0 +1,47 @@
+{
+    "name": "erlang_ls",
+    "version": "0.0.1",
+    "description": "Erlang Language Server",
+    "homepage": "https://github.com/erlang-ls/vscode#readme",
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/erlang-ls/vscode.git"
+    },
+    "bugs": {
+        "url": "https://github.com/erlang-ls/vscode/issues"
+    },
+    "main": "./client/out/client.js",
+    "scripts": {
+        "test": "echo \"Error: no test specified\" && exit 1",
+        "vscode:prepublish": "cd client && npm run update-vscode && cd .. && npm run compile",
+        "compile": "cd erlang_ls && rebar3 escriptize && cd ../client && npm run compile",
+        "watch": "cd client && npm run watch",
+        "postinstall": "cd client && npm install"
+    },
+    "author": "erlang_ls",
+    "license": "MIT",
+    "engines": {
+        "vscode": "^1.22.0"
+    },
+    "activationEvents": [
+        "onLanguage:erlang"
+    ],
+    "contributes": {
+        "languages": [
+            {
+                "id": "erlang",
+                "aliases": [
+                    "Erlang"
+                ],
+                "extensions": [
+                    ".erl",
+                    ".hrl",
+                    ".src"
```

Comment:
We could also include `.config`, but I'm not sure what would happen for config files which are not Erlang :)

Reply:
I went ahead and added `.config`, if it's detrimental it could be removed later.